src/book/Money.ts

```diff
@@ -66,10 +66,10 @@ export default class Money {
    *
    * The result is a localized string with the currency as a symbol
    */
-  format(scale = 2): string {
+  format(scale = 4): string {
     return djs.toDecimal(
       djs.transformScale(this._raw, scale),
-      ({ value, currency }) => moneyToString(Number(value), currency.code),
+      ({ value, currency }) => moneyToString(toFixed(Number(value)), currency.code),
```

Comment:
We round to two decimals (from 4). This is for cases where something is 1.9999, should become 2